src/Compiler/CMakeLists.txt

```diff
@@ -11,6 +11,21 @@ file(GENERATE
   INPUT ${CMAKE_CURRENT_BINARY_DIR}/ExternalUtil.hpp.cfg
   )
 
+file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/InitAccelerators.cpp "void InitAccelerators() {\n")  
```

Comment:
A possible alternative is implemented in PR https://github.com/onnx/onnx-mlir/pull/1227. In Accelerator.cpp there is a factory method called `Accelerator::create` which builds an accelerator according to the value of a enum passed to it.  If does so by invoking `nnpa::NNPAAccelerator::getInstance()` which is defined (strongly) in `NNPAAcclerator.cpp`.  When NNPAAcclerator.cpp is __not__ built (not configured) a weak definition is provided in `Accelerator.cpp`. When NNPAAcclerator.cpp is built, it provides a strong definition for that member function.

That approach would not need the generation via cmake of `InitAccelerators.cpp`

Reply:
@etiotto I addressed this suggestion in a previous response.

Reply:
Yep saw it.